test/ASTGen/exprs.swift

```diff
@@ -103,3 +103,18 @@ func testTrailingClsure() {
   acceptClosures {} y: { 42 }
   acceptClosures(x: {}, y: { 12 }) {}
 }
+
+func testStringLiteral(arg: Int) {
+  _ = "test"
+  _ = "foo\(arg)bar"
+  _ = "\(arg)"
+  _ = """
+    foo
+    bar\
+    baz
+    """
+  _ = """
+    foo\(arg)
+    \(arg)bar
+    """
```

Comment:
Add escaped characters tests.

Reply:
Could also add tests for things like string literals nested in interpolations (including multi-line literals) if you want